doc/api/vm.md

```diff
@@ -175,6 +175,11 @@ added: v0.3.1
   * `timeout` {number} Specifies the number of milliseconds to execute `code`
     before terminating execution. If execution is terminated, an [`Error`][]
     will be thrown.
+  * `contextName` {string} Human-readable name of the newly created context.
+    **Default:** `'VM Context i'`, where `i` is an ascending numerical index of
+    the created context.
+  * `contextOrigin` {string} [Origin][origin] corresponding to the newly
+    created context for display purposes. **Default:** `''`.
```

Comment:
Consider documenting that it's a (file?) URL?  The linked page doesn't do a great job at describing what it is and mentions things that aren't relevant (e.g. CORS.)